tools/converter/common/src/main/java/org/wso2/ei/tools/converter/common/generator/CodeGenVisitor.java

```diff
@@ -740,6 +744,11 @@ public void visit(AbortStmt abortStmt) {
     }
 
     @Override
+    public void visit(RetryStmt retryStmt) {
+
```

Comment:
Should we throw not-implimented/unsupported exception ?

Reply:
Added NotImplemtedException. https://github.com/wso2/product-ei/pull/1216